steps/jandi-message/0.1.0/step.yml

```diff
@@ -0,0 +1,117 @@
+title: Send a Jandi message
+summary: Send a [Jandi](http://bit.ly/2FiXoc5) message to a topick.
+description: |
+  Send a [Jandi](http://bit.ly/2FiXoc5) message to a topick.
+  You can setup **Incoming WebHook integration**.
+  To register an **Incoming WebHook integration** visit http://bit.ly/31Bxeeh
+
+  *Don't forget to enable this Step to **Always run** if
+  you want to get notified about failed builds as well!*
+
+  This Step is inspired from [Send a slck message step](https://github.com/bitrise-steplib/steps-slack-message).
+website: https://github.com/kimjunil/bitrise-step-jandi-message
+source_code_url: https://github.com/kimjunil/bitrise-step-jandi-message
+support_url: https://github.com/kimjunil/bitrise-step-jandi-message/issues
+published_at: 2019-06-19T12:50:42.36272+09:00
+source:
+  git: https://github.com/kimjunil/bitrise-step-jandi-message.git
+  commit: 22da79b01aabc84d7405732609f19dd830d31a11
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- notification
+toolkit:
+  go:
+    package_name: github.com/kimjunil/bitrise-step-jandi-message
+deps:
+  brew:
```

Comment:
it seems your step source code is not using `git` and `wget`, so please remove these dependencies, as `git` and `wget` installation would unnecessarily increase the build time